acnportal/acnsim/events/event.py

```diff
@@ -37,10 +71,32 @@ class PluginEvent(Event):
     """
     def __init__(self, timestamp, ev):
         super().__init__(timestamp)
-        self.type = 'Plugin'
+        self.event_type = 'Plugin'
         self.ev = ev
         self.precedence = 10
 
+    def _to_dict(self, context_dict=None):
+        """ Implements BaseSimObj._to_dict. """
+        attribute_dict, context_dict = super()._to_dict(context_dict)
+        # Plugin-specific attributes
+
+        # noinspection PyProtectedMember
```

Comment:
This comment tells linters that it's okay to access a protected member of an object in this case; e.g. self.ev._to_registry. We decided such access was okay in light of the benefit of making _to_registry etc. private to the user.